runatlantis.io/docs/server-configuration.md

```diff
@@ -979,6 +979,14 @@ Setting this to `false` can be useful in an air-gapped environment where a downl
   ```
   A token for Terraform Cloud/Terraform Enterprise integration. See [Terraform Cloud](terraform-cloud.html) for more details.
 
+### `--use-tf-plugin-cache`
+```bash
+atlantis server --use-tf-plugin-cache=false
+# or
+ATLANTIS_USE_TF_PLUGIN_CACHE=false
+```
+Set it to false if you want to disable terraform plugin cache
+
```

Comment:
Add documentation on the doc site not just here, as @GenPage sugested

Reply:
Ok, because is my first PR I was not sure how to add the documentation.
It's ok to add a new page? or do you think this can be added in an existing page?